addon-test-support/@ember/test-helpers/settled.js

```diff
@@ -203,25 +200,10 @@ const MAX_TIMEOUT = 10;
 export default function settled() {
   let options = arguments[0];
 
-  return new EmberPromise(function(resolve) {
-    // eslint-disable-next-line require-jsdoc
-    function scheduleCheck(counter) {
-      let timeout = TIMEOUTS[counter];
-      if (timeout === undefined) {
-        timeout = MAX_TIMEOUT;
-      }
-
-      futureTick(function() {
-        let settled = isSettled(options);
-        if (settled) {
-          // Synchronously resolve the promise
-          run(null, resolve);
-        } else {
-          scheduleCheck(counter + 1);
-        }
-      }, timeout);
-    }
-
-    scheduleCheck(0);
-  });
+  return waitUntil(
+    () => {
+      return isSettled(options);
```

Comment:
this could use shorthand arrow function syntax to simplify the code

Reply:
Yep, sounds good. Will update.